src/commands/deploy.js

```diff
@@ -50,14 +44,12 @@ class DeployCommand extends AbstractAuthenticatedCommand {
    * @see getEnvironment
    */
   async selectEnvironment(environments) {
+    const choices = environments.map(({ name }) => name);
     const response = await this.inquirer.prompt([{
       name: 'environment',
-      message: 'Select the environment containing the layout changes you want to deploy to production',
+      message: 'Select the environment containing the layout changes you want to deploy to the reference environment',
       type: 'list',
-      choices: environments
-        // NOTICE: Remove production since it should not be deployable on itself.
-        .filter((environment) => environment.type !== 'production')
-        .map(({ name }) => name),
+      choices,
```

Comment:
I think this should be kept as is.
We should remove the reference, since we are not able to do this now, it's better to remove the production instead of removing nothing.

Reply:
ok will rollback ;)